x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -132,10 +136,10 @@ describe('policy table', () => {
     const perPageButton = rendered.find('EuiTablePagination EuiPopover').find('button');
     perPageButton.simulate('click');
     rendered.update();
-    const fiftyButton = rendered.find('.euiContextMenuItem').at(1);
-    fiftyButton.simulate('click');
+    const numberOfRowsButton = rendered.find('.euiContextMenuItem').at(1);
+    numberOfRowsButton.simulate('click');
     rendered.update();
-    expect(namesText(rendered).length).toBe(50);
+    expect(namesText(rendered).length).toBe(25);
```

Comment:
nit: I know this was pre-existing, but the naming of `namesText()` isn't very clear.